<Type Name="AuthenticationFailureException" FullName="Microsoft.AspNetCore.Authentication.AuthenticationFailureException">
  <TypeSignature Language="C#" Value="public class AuthenticationFailureException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit AuthenticationFailureException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:Microsoft.AspNetCore.Authentication.AuthenticationFailureException" />
  <TypeSignature Language="VB.NET" Value="Public Class AuthenticationFailureException&#xA;Inherits Exception" />
  <TypeSignature Language="F#" Value="type AuthenticationFailureException = class&#xA;    inherit Exception" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.AspNetCore.Authentication.Abstractions</AssemblyName>
    <AssemblyVersion>8.0.0.0</AssemblyVersion>
    <AssemblyVersion>9.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Attributes>
    <Attribute>
      <AttributeName Language="C#">[System.Runtime.CompilerServices.Nullable(0)]</AttributeName>
      <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.Nullable(0)&gt;]</AttributeName>
    </Attribute>
    <Attribute>
      <AttributeName Language="C#">[System.Runtime.CompilerServices.NullableContext(2)]</AttributeName>
      <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.NullableContext(2)&gt;]</AttributeName>
    </Attribute>
  </Attributes>
  <Docs>
    <summary>
            A generic authentication failure.
            </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AuthenticationFailureException (string? message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.AspNetCore.Authentication.AuthenticationFailureException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="F#" Value="new Microsoft.AspNetCore.Authentication.AuthenticationFailureException : string -&gt; Microsoft.AspNetCore.Authentication.AuthenticationFailureException" Usage="new Microsoft.AspNetCore.Authentication.AuthenticationFailureException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.AspNetCore.Authentication.Abstractions</AssemblyName>
        <AssemblyVersion>8.0.0.0</AssemblyVersion>
        <AssemblyVersion>9.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">The message that describes the error.</param>
        <summary>
            Creates a new instance of <see cref="T:Microsoft.AspNetCore.Authentication.AuthenticationFailureException" />
            with the specified exception <paramref name="message" />.
            </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public AuthenticationFailureException (string? message, Exception? innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.AspNetCore.Authentication.AuthenticationFailureException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="F#" Value="new Microsoft.AspNetCore.Authentication.AuthenticationFailureException : string * Exception -&gt; Microsoft.AspNetCore.Authentication.AuthenticationFailureException" Usage="new Microsoft.AspNetCore.Authentication.AuthenticationFailureException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.AspNetCore.Authentication.Abstractions</AssemblyName>
        <AssemblyVersion>8.0.0.0</AssemblyVersion>
        <AssemblyVersion>9.0.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">The message that describes the error.</param>
        <param name="innerException">The exception that is the cause of the current exception, or <see langword="null" />.</param>
        <summary>
            Creates a new instance of <see cref="T:Microsoft.AspNetCore.Authentication.AuthenticationFailureException" />
            with the specified exception <paramref name="message" /> and
            a reference to the inner exception that is the cause of this exception.
            </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
  </Members>
</Type>
